.platform.app.yaml

```diff
@@ -1,80 +1,51 @@
-# The name of this app. Must be unique within a project.
-name: app
+# .platform.app.yaml
 
-# There is no runtime requirement, but we want the latest Node at build time.
-type: nodejs:10
-# Because there is no composer.json file we deactivate the build flavor.
-build:
-    flavor: none
+# The name of this application, which must be unique within a project.
+name: 'app'
 
-# A static site has only minimal CPU and memory needs, so don't ask for much.
-size: S
+# The type key specifies the language and version for your application.
+type: 'nodejs:12'
 
-# The documentation is built using GitBook. Install the latest version
-# of it globally.
-dependencies:
-  nodejs:
-    gitbook-cli: "2.3.0"
-
-# Use a build hook to install the GitBook plugins specified by the book.json file,
-# then build the book. The generated files will be placed in the _book directory
-# by default.
+# The hooks that will be triggered when the package is deployed.
 hooks:
-    build: |
-      set -e
-
-      # Install dependencies
-      npm install
-      npm run build
-
-      # Build example configuration files from the Registry so they can be pulled into docs.
-      node src/scripts/updateExampleConfigFiles.js
-
-      # Install and compile GitBook
-      gitbook install
-      gitbook build
-
-      # Gitbook doesn't allow for a custom favicon except through a theme plugin. So Plan B.
-      cp src/images/favicon.ico _book/gitbook/images/favicon.ico
+    # Build hooks can modify the application files on disk but not access any services like databases.
+    build: !include
+      type: string
+      path: build.sh
+    deploy: !include
+      type: string
+      path: deploy.sh
 
-      # Gitbook doesn't copy over the generated example configuration files to the built book
-      #  automatically, so we have to do that manually.
-      cp -r src/registry/images/examples _book/registry/images
-      cp -r src/registry/images/tables _book/registry/images
-      cp -r src/registry/images/registry.yaml _book/registry/images/
 
-# There is no need for a writable file mount, so set it to the smallest possible size.
-disk: 256
+# The size of the persistent disk of the application (in MB).
+disk: 5120
 
-# Configure the web server to serve our static site.
+# The configuration of the application when it is exposed to the web.
 web:
     commands:
-        # Run a no-op process that uses no CPU resources, since this is a static site.
-        start: sleep infinity
+      # Run the Meilisearch server
+      start: "./meilisearch/meilisearch --http-addr localhost:${PORT}"
```

Comment:
If the goal is for this to be the central search repository for multiple sites, I agree with Ori that it should be its own project if at all possible.  I don't know how much complexity that adds, though.  At the very least it should be its own app container.  (We aren't paying for it, so an extra app container is cheap. :smile: )

Reply:
I'm testing that now, totally agree